app/models/uk_or_overseas.rb

```diff
@@ -1,7 +1,17 @@
 class UkOrOverseas < Base
   attribute :uk_or_overseas, :string
+  attribute :country_id, :string
 
-  validates :uk_or_overseas, inclusion: { in: %w(UK overseas), message: "Select if you live in the UK or overseas" }
+  before_validation :set_country_id
+
+  validates :uk_or_overseas, inclusion: { in: %w[UK overseas], message: "Select if you live in the UK or overseas" }
+  validates :country_id, types: { method: :get_country_types }, allow_nil: true
+
+  def set_country_id
+    if uk_or_overseas == "UK"
+      self.country_id = ApiClient.get_country_types.find { |v| v.value = "United Kingdom" }.id
```

Comment:
I'd use the `id` here and whack it in a constants file somewhere (whilst not very likely, the name could change to `UK`)

Reply:
Spoke to Ian, probably a good idea, will update at next iteration